released/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.1.0.2/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "coq-mathcomp-real-closed"
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+
+homepage: "https://github.com/math-comp/real-closed"
+bug-reports: "https://github.com/math-comp/real-closed/issues"
+dev-repo: "git+https://github.com/math-comp/real-closed.git"
+license: "CeCILL-B"
+
+build: [ make "-j" "%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "coq-mathcomp-field" { (>= "1.8.0" & < "1.9.0") }
```

Comment:
Can we get the `coq` version requirements here also as in the `extra-dev` package?